recipes/clhpp/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "clhpp" %}
+{% set version = "2.0.10" %}
+{% set sha256 = "fa27456295c3fa534ce824eb0314190a8b3ebd3ba4d93a0b1270fc65bf378f2b" %}
+
+{% set OSX_VARIANT = os.environ.get("OSX_VARIANT", "native") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/KhronosGroup/OpenCL-CLHPP/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script_env:
+    - OSX_VARIANT
+
+{% if OSX_VARIANT != "native" %}
+  skip: True  # [not (osx and py36)]
+{% else %}
+  skip: True  # [unix and not py36]
+  skip: True  # [win and py<36]
```

Comment:
Requires C++11. So any Windows compiler that doesn't have support for C++11 is out. As this is not really a Python package, we don't need multiple builds with different Python versions. So Python 3.6 is basically sufficient. If newer versions of Windows compilers come out, we will support them to (as they should have C++11 support).